packages/ra-ui-materialui/package.json

```diff
@@ -0,0 +1,54 @@
+{
+    "name": "ra-ui-materialui",
+    "version": "2.0.0-beta2",
+    "description": "UI Components for react-admin with MaterialUI",
+    "files": [
+        "*.md",
+        "docs/*.md",
+        "lib",
+        "src"
+    ],
+    "main": "lib/index",
+    "authors": [
+        "François Zaninotto",
+        "Gildas Garcia"
+    ],
+    "repository": "marmelab/react-admin",
+    "homepage": "https://github.com/marmelab/react-admin#readme",
+    "bugs": "https://github.com/marmelab/react-admin/issues",
+    "license": "MIT",
+    "devDependencies": {
+        "enzyme": "~3.1.0",
+        "enzyme-adapter-react-16": "~1.0.1",
+        "file-api": "~0.10.4",
+        "full-icu": "~1.1.3",
+        "ignore-styles": "~5.0.1",
+        "react": "~16.0.0",
+        "react-dom": "~16.0.0",
+        "react-test-renderer": "~16.0.0"
+    },
+    "peerDependencies": {
+        "react": "~16.0.0 || ~16.1.0 || ~16.2.0",
+        "react-dom": "~16.0.0 || ~16.1.0 || ~16.2.0"
+    },
+    "dependencies": {
+        "autosuggest-highlight": "^3.1.1",
+        "classnames": "~2.2.5",
+        "inflection": "~1.12.0",
+        "lodash.debounce": "~4.0.8",
+        "lodash.get": "~4.4.2",
+        "lodash.isequal": "~4.5.0",
+        "lodash.set": "~4.3.2",
+        "material-ui": "1.0.0-beta.32",
+        "material-ui-icons": "~1.0.0-beta.17",
+        "prop-types": "~15.5.7",
+        "ra-core": "^2.0.0-beta2",
```

Comment:
please use tilde

Reply:
We don't use tilde for any references to react-admin packages in any of the other packages

Reply:
ok, we'll do that in another PR